nilearn/datasets/description/dosenbach_2010.rst

```diff
@@ -0,0 +1,22 @@
+Dosenbach 2010 atlas
+
+
+Notes
+-----
+160 ROIs covering much of the cerebral cortex and cerebellum.
+They were obtained from meta-analyses of fMRI activation studies
+and assigned into 6 networks according to a modularity analysis of
```

Comment:
> maybe make the type of modularity analysis explicit here, such that one does
> not need to go back read the paper

Really? I don't think that we should strive to summarize or explain
methods that we are not implementing.